This error'ed with following:
```
import: 'ESMF'                                                                                                                 
Traceback (most recent call last):                                                                                             
  File "$PREFIX/lib/python3.8/site-packages/ESMF/interface/loadESMF.py", line 127, in <module>                                 
    _ESMF = ct.CDLL(os.path.join(libsdir,'libesmf_fullylinked.so'),                                                            
  File "$PREFIX/lib/python3.8/ctypes/__init__.py", line 381, in __init__                                                       
    self._handle = _dlopen(self._name, mode)                                                                                   
OSError: /lib64/libk5crypto.so.3: undefined symbol: EVP_KDF_ctrl, version OPENSSL_1_1_1b                                       
    _ESMF = ct.CDLL(os.path.join(libsdir,'libesmf_fullylinked.so'),                                                            
  File "/scratch/v45/aph502/tmp/conda-build/esmpy_1600913455659/_test_env_placehold_placehold_placehold_placehold_placehold_plac
ehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl
aceho/lib/python3.8/ctypes/__init__.py", line 381, in __init__                                                                 
    self._handle = _dlopen(self._name, mode)                                                                                   
OSError: /lib64/libk5crypto.so.3: undefined symbol: EVP_KDF_ctrl, version OPENSSL_1_1_1b                                       

During handling of the above exception, another exception occurred:                                                            

Traceback (most recent call last):
  File "/scratch/v45/aph502/tmp/conda-build/esmpy_1600913455659/test_tmp/run_test.py", line 2, in <module>                     
    import ESMF
  File "/scratch/v45/aph502/tmp/conda-build/esmpy_1600913455659/_test_env_placehold_placehold_placehold_placehold_placehold_plac
ehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl
aceho/lib/python3.8/site-packages/ESMF/__init__.py", line 70, in <module>                                                      
    from ESMF.api.esmpymanager import *
  File "/scratch/v45/aph502/tmp/conda-build/esmpy_1600913455659/_test_env_placehold_placehold_placehold_placehold_placehold_plac
ehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl
aceho/lib/python3.8/site-packages/ESMF/api/esmpymanager.py", line 11, in <module>                                              
    from ESMF.interface.cbindings import *
  File "/scratch/v45/aph502/tmp/conda-build/esmpy_1600913455659/_test_env_placehold_placehold_placehold_placehold_placehold_plac
ehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl
aceho/lib/python3.8/site-packages/ESMF/interface/cbindings.py", line 13, in <module>                                           
    from ESMF.interface.loadESMF import _ESMF
  File "/scratch/v45/aph502/tmp/conda-build/esmpy_1600913455659/_test_env_placehold_placehold_placehold_placehold_placehold_plac
ehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl
aceho/lib/python3.8/site-packages/ESMF/interface/loadESMF.py", line 131, in <module>                                           
    raise ImportError('The ESMF shared library did not load properly.')                                                        
ImportError: The ESMF shared library did not load properly.
```

--- a/esmpy/meta.yaml
+++ b/esmpy/meta.yaml
@@ -23,7 +23,7 @@ requirements:
     host:
         - python
         - numpy
-        - krb5
+        - krb5=1.17.1
     run:
         - python
         - numpy

--- a/esmpy/meta.yaml
+++ b/esmpy/meta.yaml
@@ -23,18 +23,17 @@ requirements:
     host:
         - python
         - numpy
-        - krb5=1.17.1
+        - krb5
     run:
         - python
         - numpy
+        - openssl
 
 test:
     imports:
         - ESMF
 
 about:
-    home: http://www.earthsystemmodeling.org/esmf_releases/public/ESMF_8_0_0/esmpy_doc/html/index.html
-
     home: https://www.earthsystemcog.org/projects/esmpy/
     license: The University of Illinois/NCSA
     license_file: LICENSE

--- a/esmpy/meta.yaml
+++ b/esmpy/meta.yaml
@@ -30,6 +30,8 @@ requirements:
         - openssl
 
 test:
+    requires:
+        - openssl
     imports:
         - ESMF
 

